app/models/publication.rb

```diff
@@ -19,6 +19,10 @@ def self.publication_types
     ]
   end
 
+  def self.statuses
+    ['Published', 'In Press']
```

Comment:
You could define these as constants:

``` ruby
PUBLISHED_STATUS = 'Published'
```

And the reference them elsewhere, like in StatusMapper, as `Publication::PUBLISHED_STATUS`